spec/notifications/client/email_notification_spec.rb

```diff
@@ -48,4 +50,55 @@
         with(body: { email_address: "email@gov.uk", template_id: "f6895ff7-86e0-4d38-80ab-c9525856c3ff" })
     end
   end
+
+  describe "#send_email with a document" do
+    include Notifications::Client::HelperMethods
+
+    it "sends an email with a prepared File object" do
```

Comment:
This feels a bit weird as a test because we essentially end up comparing `prepare_upload` output with itself, which isn't giving us that much. What I'd do instead is:
* Compare with a hardcoded base64 string in the StringIO test so that we know prepare_upload does the encoding  and wraps everything in a `{"file": ...}` dictionary as expected
* For a file test what we really want to know is that the file object is read and encoded correctly, so I think we can test `prepare_upload` in this case without sending anything and make some assertions on the output (e.g. that the length is correct or that we can decode it back and match the original file contents)

Reply:
Good point - I've changed the tests. Since they're no longer really testing response of the `send_email` method and are testing the `prepare_upload` method instead, I've taken them out of this file and put them in a new file, `helper_methods_spec.rb`.